src/main/java/mops/foren/applicationservices/PostService.java

```diff
@@ -1,9 +1,6 @@
 package mops.foren.applicationservices;
 
-import mops.foren.domain.model.Permission;
-import mops.foren.domain.model.Post;
-import mops.foren.domain.model.ThreadId;
-import mops.foren.domain.model.User;
+import mops.foren.domain.model.*;
```

Comment:
Did you create this import by hand or is this the result of SaveActions?

Reply:
This was created by SA, would you change it ?

Reply:
I would only import required Objects

Reply:
Approved.